build.gradle

```diff
@@ -25,8 +25,8 @@ apply plugin: "signing"
 
 group = "com.github.adamyork"
 
-sourceCompatibility = 1.7
-targetCompatibility = 1.7
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
```

Comment:
source compatibility needs to match that of the version of wiremock its built against.

Reply:
I'll make the change.  
BTW, the current official version from Oracle is already 9. :)
and I had no issue to run both jars together.